packages/bpk-component-navigation-bar/src/BpkNavigationBar.tsx

```diff
@@ -87,13 +93,14 @@ const BpkNavigationBar = (props: Props) => {
         )}>
           <BpkText
             id={titleId}
-            textStyle={TEXT_STYLES.heading5}
+            textStyle={titleTextStyle ?? TEXT_STYLES.heading5}
```

Comment:
With the above you can then just make this

```suggestion
            textStyle={titleTextStyle}
```